src/index/Index.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "index/IndexImpl.h"
 #include "util/MemorySize/MemorySize.h"
+#include "util/http/HttpParser/AcceptHeaderQleverVisitor.h"
```

Comment:
This header seems completely out of place. Is it actually correct to include here?

Reply:
sorry to bother you. I had seen them + removed them (IDE artifact), but forgotten to push.